### DISCUSSION TITLE
Using multiple doors for alarm triggers

### DISCUSSION THREAD

**Discussion by itscelliott** [MiguelRipoll23/homebridge-securitysystem#448] [Q&A]
Hey everyone! This may be a dumb question but here it is...

I have 4 door sensors at my house. I am trying to setup a system to...

1. Arm security system at sunset - DONE
2. If any of 4 doors are opened, trigger the NIGHT TRIGGER - NEED HELP HERE 
3. Once triggered, have an automation play siren sounds on my Apple HomePod - DONE

Basically my issue here is that I cannot find a way to get an automation to look for all 4 doors at the same time, I can only find a way to call one at a time and I dont want 500 automations clogging up my view =D

Anyone have any idea on how to do this via this Plugin, another, or natively? 

Thanks in advance!

**Comment by MiguelRipoll23** [comment]
Hey there,

If you use the Eve app, you can add multiple triggers to a single automation (be aware that they are called rules inside that app).

That will help you out with your automations setup.
